django/thunderstore/repository/models/uploader_identity.py

```diff
@@ -87,3 +87,18 @@ def can_user_upload(self, user):
             UploaderIdentityMemberRole.owner,
             UploaderIdentityMemberRole.member,
         )
+
+    def can_create_service_account(self, user) -> bool:
```

Comment:
Let's make this function describe it's reasons for failing a permission check, e.g. by making it raise a validation error, or just return the error message(s) in the return value.